CHANGELOG.md

```diff
@@ -15,6 +15,10 @@ v0.23 + 1
 
 * `git_cert` descendent types now have a proper `parent` member
 
+* It is the responsibility fo the refdb backend to decide what to do
+  with the reflog on ref deletion. The file-based backend must delete
+  it, a database-backed one may wish to archive it.
```

Comment:
Would this last option be chosen, after th reference is deleted, the backend should still return still `false` when `git_reference_has_log()` is called.